tests/assets/sysext/README.md

```diff
@@ -0,0 +1,35 @@
+This folder contains 2 sysextensions
+
+work.raw contains a simple script called `hello.sh` that prints "Hello World" to the console.
+hello-broke.raw contains a simple script called `hello.sh` that prints "Hello World" to the console, but it is NOT signed or verity checked.
```

Comment:
Maybe it's a good idea to have the "bad" extension print a different text, to be sure we got the output for the "good" extensions. Maybe "Hello evil world" ?

Reply:
Checks are done in a different way to not have to relay on the output of the script. Like check if it was copied, check if it was loaded and such.